envrc.el

```diff
@@ -63,6 +62,7 @@
 (require 'ansi-color)
 (require 'cl-lib)
 (require 'inheritenv)
+(require 'tramp)
```

Comment:
I'd imagine that `(eval-when-compile (require 'tramp))` might be sufficient here btw, to possible avoid loading tramp eagerly.

Reply:
Fixed!